### DISCUSSION TITLE
pip cant install tfc

### DISCUSSION THREAD

**Discussion by Lte-DL** [tensorflow/compression#175] [Q&A]
i use pycharm and anaconda with win 11.i cont use pip to install tfc . it told me :
ERROR: Could not find a version that satisfies the requirement tensorflow-compression (from versions: none)
ERROR: No matching distribution found for tensorflow-compression
how can i solve it? if you have any maners ,please tell me, thanks

**Comment by jonaballe** [comment]
Hello, we don't currently support Windows binaries (see https://github.com/tensorflow/compression#installation). Have you tried using Docker or WSL2?

**Reply by Lte-DL** [reply]
Thank you for your replaying. I've already installed Linux for it .
